tools/cpp/windows_cc_configure.bzl

```diff
@@ -426,12 +426,12 @@ def find_llvm_path(repository_ctx):
 
     # 3. Check default directories for LLVM installation
     auto_configure_warning_maybe(repository_ctx, "Looking for default LLVM installation directory")
-    program_files_dir = _get_path_env_var(repository_ctx, "PROGRAMFILES")
+    program_files_dir = _get_path_env_var(repository_ctx, "ProgramFiles")
```

Comment:
@gjasny Can you instead make this function(`_get_path_env_var`) case-insensitive?